config/image-relocation-webhook/400-crds.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: imagemaps.webhook.image-relocation.pivotal.io
+spec:
+  group: webhook.image-relocation.pivotal.io
+  version: v1alpha1
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+  scope: Cluster
+  names:
+    plural: imagemaps
+    singular: imagemap
+    kind: ImageMap
```

Cluster scoped resources, should be use `Cluster` as a prefix

```suggestion
    kind: ClusterImageMap
```

---

Adopted.